Userland/Applications/Presenter/PresenterWidget.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "LibGUI/Action.h"
+#include "Presentation.h"
+#include <LibGUI/Event.h>
+#include <LibGUI/UIDimensions.h>
+#include <LibGUI/Widget.h>
+
+// Title, Author
+constexpr StringView const title_template = "{} ({}) — Presenter"sv;
+
+class PresenterWidget : public GUI::Widget {
+    C_OBJECT(PresenterWidget);
+
+public:
+    PresenterWidget();
+    ErrorOr<void> initialize_menubar();
+
+    virtual ~PresenterWidget() = default;
```

Comment:
`virtual ~PresenterWidget() override = default;`
or remove this, widget already has a polymorphic destructor.

Reply:
This is kinda standard practice? The override needs to be present though, yes.